playbooks/02 - Use Case - FortiSOAR for Microsoft Teams/Enrich IP > Enrichment.json

```diff
@@ -0,0 +1,569 @@
+            "name": "Get Rep IPStack -> Formatted Response",
+            "targetStep": "\/api\/3\/workflow_steps\/51b08fdd-056e-44a5-bc00-8b060edeeea4",
+            "sourceStep": "\/api\/3\/workflow_steps\/f926c60a-309f-4f35-baea-5822464e6d34",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "21de1988-f4a1-431d-a4f9-a01616a6e781"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Is Indicator Available -> Create New Indicator",
+            "targetStep": "\/api\/3\/workflow_steps\/5155d099-b1ce-4836-82e4-dd8e83a3487a",
+            "sourceStep": "\/api\/3\/workflow_steps\/f1e0ce6e-e9df-48af-b831-0b5d0de0bdf5",
+            "label": "No",
+            "isExecuted": false,
+            "group": null,
+            "uuid": "2ac0e782-18da-480f-93d1-56448e07ac8c"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Is Indicator Available -> find Record",
+            "targetStep": "\/api\/3\/workflow_steps\/f594ad26-be2b-4b1a-b529-63c738cc2fc1",
+            "sourceStep": "\/api\/3\/workflow_steps\/f1e0ce6e-e9df-48af-b831-0b5d0de0bdf5",
+            "label": "Yes",
+            "isExecuted": false,
+            "group": null,
+            "uuid": "4104d9fa-8c73-4901-a9e0-27af934beb67"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Is Indicator Present -> Is Indicator Available",
+            "targetStep": "\/api\/3\/workflow_steps\/f1e0ce6e-e9df-48af-b831-0b5d0de0bdf5",
+            "sourceStep": "\/api\/3\/workflow_steps\/41b4bf89-b181-43a5-8c6a-ddfc24575be5",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "90da9b57-87ac-4168-a6eb-58ecd0deb2ec"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Parameters Required -> Response Step",
+            "targetStep": "\/api\/3\/workflow_steps\/1332ad1c-4523-48ee-95df-fb580df228b1",
+            "sourceStep": "\/api\/3\/workflow_steps\/ff3ea299-1ae7-43f1-a862-447c43836c2c",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "28ab2859-cdf6-436e-bdab-2be84e6ab7a1"
+        },
+        {
+            "@type": "WorkflowRoute",
+            "name": "Start -> Are Parameters Passed",
+            "targetStep": "\/api\/3\/workflow_steps\/d5ff65e3-503b-4b43-895d-40447ae149ba",
+            "sourceStep": "\/api\/3\/workflow_steps\/26f4fb12-6799-4b1e-98e3-21e019e634f3",
+            "label": null,
+            "isExecuted": false,
+            "group": null,
+            "uuid": "7a7982bd-9f90-451e-8085-72431d20cc52"
+        }
+    ],
+    "groups": [],
+    "priority": "\/api\/3\/picklists\/2b563c61-ae2c-41c0-a85a-c9709585e3f2",
+    "uuid": "9e99ba9b-1536-444e-8aae-db904485567c",
+    "owners": [],
+    "isPrivate": false,
+    "deletedAt": null,
+    "recordTags": [
+        "bot_enabled"
```

Comment:
referenced also needs to be present as a standard practice. Please check few PBs in SFSP